src/legacy/components/ConfirmationDialog.jsx

```diff
@@ -70,11 +70,12 @@ const ConfirmationDialog = ({
     okButtonText,
     cancelButtonText,
     isOkButtonEnabled,
+    dimBackground,
```

Comment:
Is this ever true?

Reply:
Yes, it is used here in the same PR: https://github.com/NordicSemiconductor/pc-nrfconnect-launcher/blob/d2e99bde4184abda3c823922a7229f775a13a61b/src/legacy/app/components/LegacyAppDialog.jsx#L44

Reply:
Yes, that was what I was wondering about. I now learned that not passing a value to a prop defaults to passing true, so now I get it 👍